helm-charts/baremetal-operator/templates/baremetal-operator.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
You don't need template for namespace specifically. That can be done with `helm` CLI. i.e. `helm install --create-namespace --namespace baremetal-operator-system ...`

Reply:
Did not think so, but was in kustomize output. I will delete once you are done with initial comments.